src/bundle/Resources/public/scss/_header.scss

```diff
@@ -3,6 +3,7 @@
 
     .container {
     	position: relative;
+        overflow: auto;
```

Comment:
```suggestion
       overflow: auto;
```

Reply:
Discussed with @pawbuj - Github is showing me additional space, which does not exist in code.
Everything is OK. :)